runtime/drivers/drivers.go

```diff
@@ -157,3 +157,23 @@ func NewPermissionDeniedError(msg string) error {
 func (e *PermissionDeniedError) Error() string {
 	return e.msg
 }
+
+type OlapCacheCtrl struct {
+	UseCache      bool
+	PopulateCache bool
+}
```

Comment:
1. As a Druid-specific setting, it should belong entirely in the `druidsqldriver` package, not pollute the `runtime/drivers` package. The `druidsqldriver` package is also supposed to be a generic `database/sql` driver so having a dependency on `runtime/drivers` is odd.
2. Did you consider passing it as a query argument instead and implementing the `NamedValueChecker` interface? See the docs [here](https://pkg.go.dev/database/sql/driver#NamedValueChecker). This is the recommended way to pass query-level settings through the `database/sql` interface.
3. This code definitely needs a docstring – without context, it's very unclear what it does and what caches it relates to.